_sections/admin-android-studio.md

```diff
@@ -30,6 +30,12 @@ As an administrator, you are required to install the GitLive app on the organiza
 
 ### Step 4
 
+* You will also have the option to add an issue tracker to GitLive. If you choose not to connect an issue tracker, you can do this at a later date through the settings menu. You can connect to the issue tracker attached to your repository hosting service or to Jira. In this case, since GitLive was set up on a Github organisation, the options are to slect Github issues or Jira.
```

Comment:
After succesfull installation on your repositories, you will..
You have the option to connect Jira or the issue tracker from your respective repository hosting service. In this case..

finish with 
if you choose not to connect ....  stage better then date?
gitlive settings in your ide